docs/rpc_package.md

```diff
@@ -98,6 +98,18 @@ console.log('-- Packed data:', packedData);
 
 You can simulate the `PACK` instruction from Michelson with the `packData` method.
 
+```js
+// runs view
```

Comment:
I suggest adding a note that this function is the execute tzip4 views (refer as lambda views) and maybe add a link to the tzip4: https://gitlab.com/tezos/tzip/-/blob/master/proposals/tzip-4/tzip-4.md#view-entrypoints

To avoid confusing between all kind of views (on-chain views, off-chain views, lambda-views)